src/ducks/categorization/services.js

```diff
@@ -186,6 +204,37 @@ const getLocalClassifierOptions = transactionsWithManualCat => {
   }
 }
 
+export const atomicReweight = (classifier, category, word, frequencyCount) => {
+  const newFrequencyCount = 1 + Math.log(frequencyCount)
+  const deltaFrequencyCount = frequencyCount - newFrequencyCount
+  // update the right entries of the classifier's parameters
+  classifier.vocabulary[word] -= deltaFrequencyCount
+  classifier.wordCount[category] -= deltaFrequencyCount
```

Comment:
Is it normal that we are substracting a frequency from a count ?

Reply:
Frequency in the statistical meaning : it's a count. Will double-check it anyway